test/unit-test/ConfigFiles/FreeRTOSIPConfig.h

```diff
@@ -300,11 +297,20 @@ extern uint32_t ulRand();
 #define ipconfigSOCKET_HAS_USER_WAKE_CALLBACK    ( 1 )
 #define ipconfigUSE_CALLBACKS                    ( 0 )
 
+#define ipconfigUSE_NBNS                         ( 1 )
 
-#define portINLINE
+#define ipconfigUSE_LLMNR                        ( 1 )
+
+#define ipconfigDNS_USE_CALLBACKS                1
+#define ipconfigUSE_ARP_REMOVE_ENTRY             1
+#define ipconfigUSE_ARP_REVERSED_LOOKUP          1
+
+#define ipconfigETHERNET_MINIMUM_PACKET_BYTES    ( 200 )
 
-void vApplicationMQTTGetKeys( const char ** ppcRootCA,
-                              const char ** ppcClientCert,
-                              const char ** ppcClientPrivateKey );
+#define ipconfigARP_STORES_REMOTE_ADDRESSES      ( 1 )
+
+#define ipconfigARP_USE_CLASH_DETECTION          ( 1 )
+
+#define portINLINE
```

Comment:
this should be defined in a port or portmacro.h
It should be part of the port layer for the kernel port in use (hence the 'port' prefix).  Adding it here is really dangerous because it means our demo will build but end user code won't

Reply:
Done.